frontend/pages/AddressBook/Overview/components/AddressList/connector.js

```diff
@@ -9,6 +10,7 @@ import { setDefaultAddress } from '../../../../../action-creators/addressBook';
 const mapStateToProps = state => ({
   addresses: getUserAddresses(state) || [],
   defaults: getUserDefaultAddresses(state) || {},
+  defaultTags: getConfig(state).splitDefaultAddressesByTags,
```

Comment:
`splitDefaultAddressesByTags` should be renamed as suggested in the comment within the `shopgate.user.getConfig.v1` pipeline definition